sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/Capabilities.java

```diff
@@ -37,6 +56,10 @@ public boolean supportsCNINetworking() throws IOException {
         return hasOrExceedsVersion(1, 9);
     }
 
+    public boolean supportsPreReservedResources() {
+        return false;
```

Comment:
I plan to have another PR, with the real released Mesos bits and at that point we'll determine what the best way is to determine whether support is enabled.  Not sure what Mesos has planned yet.  Is it a capability on Mesos endpoint or is it just based on DC/OS version?